src/routes/api/collabs/[id]/picks/[pick_id]/index.ts

```diff
@@ -212,6 +262,7 @@ export const del: RequestHandler = async ({ request, params }) => {
 				pick.collab.status === CollabStatus.EARLY_ACCESS
 			) {
 				await deletePick(pick);
+				sendEmbedToDiscord({ pick, user, reason });
```

Comment:
This path covers a user deleting their own pick, no need to notify the user of this, since they're the person that deleted their own pick. Take this into account in the frontend as well, so the user doesn't need to provide a reason to delete their own pick.

Reply:
Removed it and added a check on the frontend for the reason so it doesn't trigger.